pages/core-concepts/seed.mdx

```diff
@@ -22,17 +22,64 @@ Here's why it's beneficial:
 
 While generated data might be the unsung hero of early development, it can get a bit needy as software evolves. But don't sweat it! Our mission is to save you from the never-ending saga of maintaining those seed scripts. Who has time for that, right?
 
-## Introducing `@snaplet/seed`
+## The workflow at a glance
 
-The key to effortless generated data is a tool that deeply understands your database's schema. By introspecting your database, we are able to create a fully-typed client dedicated to data generation.
```

Comment:
It felt like (2) below accomplished the job of this paragraph, so I removed this, making sure I'd included over there what seemed like the important points over here.